apps/core/scenes/search/SearchFormModes.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+
+import * as React from 'react';
+import { View, Platform } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { StyleSheet } from '@kiwicom/universal-components';
+import {
+  PassengersButton,
+  TripTypeButton,
+  TripTypeSwitch,
+} from '@kiwicom/margarita-components';
+import {
+  withLayoutContext,
+  LAYOUT,
+  type LayoutContextState,
+} from '@kiwicom/margarita-utils';
+
+import {
+  withSearchContext,
+  type SearchContextState,
+  type TripTypes,
+  type ModalTypes,
+} from './SearchContext';
+import { MODAL_TYPE, TRIP_TYPE } from './SearchConstants';
+
+type Props = {|
+  +tripType: string,
+  +adults: number,
+  +infants: number,
+  +bags: number,
+  +setTripType: TripTypes => void,
+  +setModalType: ModalTypes => void,
+  +layout: number,
+|};
+
+class SearchFormModes extends React.Component<Props> {
+  handleTripTypeSelect = (type: string) => {
```

Comment:
Do we need to use the switch for something?
Why not just the `if` condition?

Reply:
Not really. Maybe only thing is that this condition can grow if we add more trip types later.
But for now `if` is definitely shorter and we can refactor it later if it grow bigger with other types.